### PR TITLE
Change all multi-line textboxes to use the GOV.UK Frontend textarea component

### PR DESCRIPTION
**Pull request by kr8n3r**
Fixes: https://trello.com/c/wz4oz7EA/741-change-all-multi-line-textboxes-to-use-the-govuk-frontend-textarea-component

Replace the use of our custom textbox macro with the GOV.UK Design System Textbox component.

See individual commits for details. 

Run locally and check affected routes, like
 - edit/create template
 - platform-admin service, organisation and billing notes and returned letters
 - add letter contact
 - send one-off letter
 - support feedback form

### CODE REVIEW

File: app/main/forms.py [hunk 1]
```diff
@@ -300,6 +300,36 @@ class GovukSearchField(GovukTextInputFieldMixin, SearchField):
     param_extensions = {"classes": "govuk-!-width-full"}
 
 
+class GovukTextareaField(GovukFrontendWidgetMixin, TextAreaField):
+    govuk_frontend_component_name = "textarea"
+
+    def get_error_message(self, error_message_format="text"):
```
Comment (tombye) [inline app/main/forms.py:306]: Think this is inherited from `GovukFrontendWidgetMixin` so doesn't need defining again.

**Review by tombye — CHANGES_REQUESTED** [review]
This all looks good to me and only a few comments. Happy to approve once they've been looked at.

File: app/assets/javascripts/enhancedTextbox.js [hunk 2]
```diff
@@ -18,12 +18,17 @@
         !!textarea.data('highlightPlaceholders')
       );
 
+      this.autofocus = (
+        Boolean(textarea.data('autofocusTextbox')) ||
```
[the remark this answers is not in the document]
Comment (kr8n3r) [reply app/assets/javascripts/enhancedTextbox.js:22]: sure

File: tests/javascripts/enhancedTextbox.test.js [hunk 3]
```diff
@@ -375,130 +342,201 @@ describe('Enhanced textbox', () => {
...
+      helpers.triggerEvent(window, 'resize');
+
+      expect(window.getComputedStyle(textarea).width).toEqual("500px");
+      expect(window.getComputedStyle(backgroundEl).width).toEqual("500px");
+
+    });
+
+    describe("If a new variable is added to the textbox, it should also be added to the element below in a highlight tag", () => {
+
+      test("If the textbox is a <textarea>", () => {
```
[the remark this answers is not in the document]
Comment (kr8n3r) [reply tests/javascripts/enhancedTextbox.test.js:468]: yep can do

**Review by tombye — APPROVED** [review]
Thanks for addressing the comments. All looking good now so 👍🏻 

Massive piece of work too, congrats!

**Pull request merged**